### PR TITLE
feat(controls): Add React controls for Image360

### PR DESCRIPTION
**Pull request by ConradJChan**
TODO:
- [x] unit tests

### CODE REVIEW

File: src/lib/viewers/box3d/image360/Image360Viewer.js [hunk 1]
```diff
@@ -30,9 +33,46 @@ class Image360Viewer extends Box3DViewer {
...
+        super.handleSceneLoaded();
+
+        this.renderUI();
+    }
+
+    /**
+     * @inheritdoc
+     */
+    handleShowVrButton() {
+        if (this.controls && this.getViewerOption('useReactControls')) {
```
Comment (jstoffan) [inline src/lib/viewers/box3d/image360/Image360Viewer.js:55]: Should the `this.controls` identity check be a guard clause? Otherwise the `else` clause below could throw.

File: src/lib/viewers/box3d/model3d/__tests__/Model3DControlsNew-test.tsx [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 import { shallow, ShallowWrapper } from 'enzyme';
-import AnimationControls from '../../../controls/model3d/AnimationControls';
+import AnimationControls from '../../../controls/box3d/model3d/AnimationControls';
 import FullscreenToggle from '../../../controls/fullscreen';
 import Model3DControls, { Props } from '../Model3DControlsNew';
-import Model3DSettings, { CameraProjection, RenderMode } from '../../../controls/model3d/Model3DSettings';
-import ResetControl from '../../../controls/model3d/ResetControl';
-import VrToggleControl from '../../../controls/model3d/VrToggleControl';
+import Model3DSettings, { CameraProjection, RenderMode } from '../../../controls/box3d/model3d/Model3DSettings';
+import ResetControl from '../../../controls/box3d/model3d/ResetControl';
```
Comment (jstoffan) [inline src/lib/viewers/box3d/model3d/__tests__/Model3DControlsNew-test.tsx:7]: Do we need the `model3d` subdirectory or can everything live in the `box3d` directory?

File: src/lib/viewers/controls/box3d/model3d/__tests__/Model3DSettings-test.tsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { shallow, ShallowWrapper } from 'enzyme';
 import Model3DSettings, { CameraProjection, Props, RenderMode } from '../Model3DSettings';
-import Settings, { Menu } from '../../settings';
+import Settings, { Menu } from '../../../settings';
```
Comment (jstoffan) [inline src/lib/viewers/controls/box3d/model3d/__tests__/Model3DSettings-test.tsx:4]: Should we do this rename/move as a separate PR?

File: src/lib/viewers/box3d/model3d/__tests__/Model3DControlsNew-test.tsx (hunk 2 above)
Comment (ConradJChan) [reply src/lib/viewers/box3d/model3d/__tests__/Model3DControlsNew-test.tsx:7]: You're right I don't think the subdirectories are needed

**Review by jstoffan — APPROVED** [review]
